src/org/labkey/test/tests/list/CrossFolderListTest.java

```diff
@@ -78,8 +171,8 @@ public void testAddDataInSubfolderToTopLevelList() throws Exception
         var displayedData = subFolderListPage.getGrid().getTableData();
         assertEquals("expect only subfolder data to be shown here by default",
                 rowsToInsert.size(), displayedData.size());
-        assertThat("expect only the data inserted here to be shown",
-                subFolderListPage.getGrid().getColumnDataAsText("String Column"), hasItems("stringy", "chewy"));
+        assertThat(subFolderListPage.getGrid().getColumnDataAsText("String Column"))
+                .as("expect only the data inserted here to be shown").contains("stringy", "chewy");
```

Comment:
Should check for an exact match.
```suggestion
        assertThat(subFolderListPage.getGrid().getColumnDataAsText("String Column"))
                .as("expect only the data inserted here to be shown").containsExactly("stringy", "chewy");
```

Reply:
My concern about that is that other test cases might create lists that will appear in this view; I only wanted to ensure that these two (that I know must be present) are there